packages/replay-next/src/contexts/PointsContext.tsx

```diff
@@ -96,7 +96,9 @@ export function PointsContextRoot({ children }: PropsWithChildren<{}>) {
         badge: null,
         content: "",
         condition: null,
+        createdByUserId: currentUserInfo?.id ?? null,
```

Comment:
`user_id` would be more consistent with our DB schema which we could transform on the JS side if we'd like

Reply:
I think the only thing that interacts with this would be the client-side JS, so I have a slight preference to follow the client side naming conventions (and transfer before sending the server, if need be)